### PR TITLE
Use flask to serve ISIC and Kaggle images

### LINKED ISSUE #12: Create route to serve images from ISIC
Child issue of #5

### PR DESCRIPTION
**Pull request by Huy-Ngo**
Clone of #13 

In this PR, I:

- create a route to serve images hosted on this server
- create a route to serve information for the images from Kaggle API
- create a route to serve information for the images from ISIC API

This will resolve #12

### CODE REVIEW

**Comment by Huy-Ngo** [comment]
image
Wait Kaggle also take from ISIC

File: backend/static_crawler.py [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+
+    if validate():
+        dataset = 'dinhanhx/skin-cancer-mnist-ham10000'
+        path = Path('./static/kaggle')
+        dataset_path = download(dataset, path)
+        scm = SkinCancerMnist(dataset_path)
+        for i in range(scm.get_len_dataset()):
+            full_data.append(scm.get_img_metadata(i))
+
+    return full_data
```
Comment (dinhanhx) [inline backend/static_crawler.py]: Parameter `n_images` is never used. I suggest you to add an if else statement to check whether it's bigger than the length of SkinCancerMnist dataset. If it's bigger, set `n_images` equal to the length.

**Review by dinhanhx — CHANGES_REQUESTED** [review]

File: backend/static_crawler.py (hunk 1 above)
Comment (Huy-Ngo) [reply backend/static_crawler.py]: Oh I copied from the isic one. I'll delete it.

**Comment by Huy-Ngo** [comment]
@dinhanhx Is it ok now?

**Comment by dinhanhx** [comment]
> @dinhanhx Is it ok now?

I think it's fine now.

**Review by dinhanhx — APPROVED** [review]

**Pull request merged**